iommi/part.py

```diff
@@ -102,9 +102,7 @@ def bind(self, *, parent=None, request=None):
             return None
         del self
         bind_members(result, name='endpoints')
-        bind_members(result, name='assets')
-        # This is necessary to force the members to actually bind
-        result.assets.items()
+        bind_members(result, name='assets', lazy=False)
```

Comment:
Not sure that this (lazy=False) is actually still necessary know that we do the early content rendering.  I think this was a late night change by myself when my brain had almost understood the actual problem.

Reply:
Hmm.. yea, if I remove that lazy param everything works the same and all tests pass.

Reply:
Then I'd say we drop the lazy parameter. If the tests pass without it it is dead or we should fix the tests :)

Reply:
done